nginx/default.conf

```diff
@@ -61,15 +61,15 @@ proxy_set_header X-Forwarded-Ssl $proxy_x_forwarded_ssl;
 proxy_set_header X-Forwarded-Port $proxy_x_forwarded_port;
 
 # carleton.api.frogpond.tech
-upstream carleton.api.frogpond.tech {
+upstream carleton-backend {
   server carleton:80;
 }
 
 server {
   server_name carleton.api.frogpond.tech;
 
   location / {
-    proxy_pass http://carleton.api.frogpond.tech;
+    proxy_pass http://carleton_backend;
```

Comment:
fun fact: underscores are illegal in hostnames. 

Most everything accepts them, though.